WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopover.swift

```diff
@@ -111,10 +111,7 @@ private enum Strings {
 
     static let subtitle = NSLocalizedString(
         "compliance.analytics.popover.subtitle",
-        value: """
-                We process your personal data to optimize our website and
-                marketing activities based on your consent and our legitimate interest.
-                """,
+        value: "We process your personal data to optimize our website and marketing activities based on your consent and our legitimate interest.",
```

Comment:
@mokagio Will this reflect in `Localizable.strings` before release or should we update it manually?